test/functional/cypress/integration/google/accelerator_zones.spec.js

```diff
@@ -7,23 +7,28 @@ describe('google: Create Server Group Modal GPU Accelerators', () => {
 
   beforeEach(() => {
     registerDefaultFixtures();
-    cy.route('/credentials?expand=true', 'fixture:google/accelerator_zones/credentials.json');
-    cy.route('/images/find?*', 'fixture:google/shared/images.json');
+    cy.intercept('/credentials?expand=true', {
+      fixture: 'google/accelerator_zones/credentials.json',
+    });
+
+    cy.intercept('/images/find?*', {
+      fixture: 'google/shared/images.json',
+    });
   });
 
   it('should not show NVIDIA Tesla K80 initially', () => {
     cy.visit('#/applications/compute/clusters');
     cy.window().its('angular').should('exist');
-    cy.get('button:contains("Create Server Group")', { timeout: 10000 })
-      .should('be.visible')
-      .click();
+    cy.get('button:contains("Create Server Group")', { timeout: 10000 }).should('be.visible').click();
+
+    cy.contains('ul.steps-indicator li a', 'Advanced Settings').click();
```

Comment:
New check looks like?  I like if I'm reading the syntax right :)

Reply:
Yes, the original test assumed Cypress would automatically scroll the element (the button clicked just below) into the viewport. But in newer Cypress versions, the default scroll behavior doesn’t always bring elements fully into view before interacting with them.

This used to be handled by a `scrollBehavior.js` support script that forced elements to scroll to the center of the viewport:

```js
Cypress.on('scrolled', $el => {
  $el.get(0).scrollIntoView({ block: 'center', inline: 'center' });
});
```

That script was removed in this PR and partially replaced by setting `scrollBehavior: 'center'` in Cypress config, which now achieves the same effect more cleanly.